Trimble.Modus.Components/Controls/Button/TMButton.xaml.cs

```diff
@@ -167,15 +171,19 @@ private static void OnIsDisabledChanged(BindableObject bindable, object oldValue
 
     private static void OnButtonColorChanged(BindableObject bindable, object oldValue, object newValue)
     {
-        if (bindable is TMButton tmButton && !tmButton.IsFloatingButton)
+        if (bindable is TMButton tmButton)
         {
             CheckButtonStyle(tmButton);
         }
     }
 
     private static void CheckButtonStyle(TMButton tmButton)
```

Comment:
Can you rename this method? `CheckButtonStyle` doesn't seem right. What about `UpdateButtonStyle`?

Reply:
Updated